azure-pipelines/build_stage.windows.yml

```diff
@@ -2,12 +2,40 @@ parameters:
   - name: stage_name
     type: string
     default: ''
+  - name: packages
+    type: string
+    default: ''
 
 steps:
 - powershell: |
     if ("${{ parameters.stage_name }}") {
-      .\gitlab-ci\build_stage.ps1 "${{ parameters.stage_name }}" "${Env:Agent_TempDirectory}/roaster-scratch"
+      .\gitlab-ci\build_stage.ps1 "${{ parameters.stage_name }}"  "${Env:Agent_TempDirectory}/roaster-scratch"
```

Comment:
Recommend to use / for path.
\ can often be treated as escape, and won't work in anywhere outside windows.

Reply:
yes. but this is powershell task. pwsh is also cross-platform.

Reply:
Will powershell treats `a\n` as `a<newline>` or `a/n` elsewhere?

Reply:
we're talking about the path separator here. and powershell is using ` to as escape character